### PR TITLE
Import timestamps

### PR DESCRIPTION
**Pull request by FrittenKeeZ**
- Assign file last modified as timestamps during import
- Added microsecond precision to form submissions

### CODE REVIEW

File: src/Globals/GlobalSet.php [hunk 1]
```diff
@@ -59,9 +57,4 @@ public function model($model = null)
 
         return $this;
     }
-
-    public function path()
```
Comment (ryanmitchell) [inline src/Globals/GlobalSet.php:63]: This shouldnt be removed, I added it to get around issues when extending the stache globals

**Comment by FrittenKeeZ** [comment]
@ryanmitchell This should do it for getting timestamps from files - I also refactored a few things regarding progress bars

**Comment by ryanmitchell** [comment]
Looks good apart from that path() change.
Thanks for all the work on this!

File: src/Globals/GlobalSet.php (hunk 1 above)
Comment (FrittenKeeZ) [reply src/Globals/GlobalSet.php:63]: @ryanmitchell what issue is that? Is there a way to replicate it?

**Comment by ryanmitchell** [comment]
We need to override the path() method as it won’t be returning the right value. I can’t remember the exact error now but I changed it because it was needed.

**Comment by FrittenKeeZ** [comment]
> We need to override the path() method as it won’t be returning the right value. I can’t remember the exact error now but I changed it because it was needed.
> […](#)
> On 11 Aug 2022, at 15:22, Frederik Sauer ***@***.***> wrote: ﻿ @FrittenKeeZ commented on this pull request. In src/Globals/GlobalSet.php: > @@ -59,9 +57,4 @@ public function model($model = null) return $this; } - - public function path() @ryanmitchell what issue is that? Is there a way to replicate it? — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you were mentioned.

I don't experience any errors either in CP or on the site with path not being overridden.

**Comment by FrittenKeeZ** [comment]
@ryanmitchell Maybe it was needed a few months ago when you started?

**Comment by ryanmitchell** [comment]
Maybe. That syncOriginal() is strange, it was there before so must have been removed as part of the refactor. The code I was looking at locally had it so I wasnt getting the error...

**Comment by ryanmitchell** [comment]
Yeah it was removed here: 1f3282906859b6b49fe241b148926babdb4bcd0a

**Comment by FrittenKeeZ** [comment]
> Yeah it was removed here: [1f32829](https://github.com/ryanmitchell/eloquent-driver/commit/1f3282906859b6b49fe241b148926babdb4bcd0a)

Ahhh, I must've copy/pasted wrongly... but NavTree needed the fix as well as Term

**Pull request merged**